### PR TITLE
Update esdt-tokens.mdx

### PR DESCRIPTION
**Pull request by juliancwirko**
Add info about optional method call also in ESDT multi transfer

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [x] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/tokens/esdt-tokens.mdx [hunk 1]
```diff
@@ -330,6 +330,8 @@ Sending a transaction containing both an ESDT transfer _and a method call_ allow
 There is also the possibility to perform multiple tokens transfers in a single bulk. This way, one can send (to a single receiver) multiple
 fungible, semi-fungible or non-fungible tokens via a single transaction.
 
+It is also possible to send ESDT tokens to a contract as part of a method call.
```
Comment (bogdan-rosianu) [inline docs/tokens/esdt-tokens.mdx:333]: maybe define a new section for multi transfers with sc calls? similar to this one: 
https://docs.multiversx.com/tokens/esdt-tokens#transfers-to-a-smart-contract

Comment (juliancwirko) [reply docs/tokens/esdt-tokens.mdx:333]: Sure, I thought about it too.
Something like `Multiple tokens transfer to a smart contract` without copying the general information (like: Payable versus non-payable) from `Transfers to a smart contract` with new code example, right?

Comment (bogdan-rosianu) [reply docs/tokens/esdt-tokens.mdx:333]: I think those information can be copied as well. Even if the info is duplicated, it's more intuitive if one receives a link to that docs section. Also, place it after the multi tokens transfer definition

Comment (juliancwirko) [reply docs/tokens/esdt-tokens.mdx]: added a new section

**Review by bogdan-rosianu — APPROVED** [review]

File: docs/tokens/esdt-tokens.mdx (hunk 1 above)
Comment (bogdan-rosianu) [reply docs/tokens/esdt-tokens.mdx]: 🚀

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**